frontend/src/App.js

```diff
@@ -8,6 +8,7 @@ import { ViewUsers } from './views/viewUsers';
 import { Paper } from '@mui/material';
 import { BCTypography } from 'components/BCTypography';
 import UserTabPanel from 'layouts/admin/UserTabPanel';
+import { EditUser } from './views/editUser';
```

Comment:
It doesnt make much sense to repeat our work ( the layout is the same other than a couple fields ). It would be more beneficial in my opinion set a flag for what fields to show as is already done in `EditUser.jsx`. 

The folders dont necessarily have to reflect the route structure, as that is already done via the routes object that gets passed into react router. This is one of the questions i was planning to ask the devs for our upcoming tooling and standards update.

For now as we already have the pieces to switch between idir and bceid user views within the page component, I think we can leave this for now unless there are bigger conerns.

Thoughts?